toolkit/docs/formats/imageconfig.md

```diff
@@ -337,6 +337,11 @@ The Security Enhanced Linux (SELinux) feature is enabled by using the `SELinux`
 This will instruct init (systemd) to set the configured mode on boot.  The `force_enforcing` option will set enforcing in the config and also add `enforcing=1` in the kernel command line,
 which is a higher precedent than the config file. This ensures SELinux boots in enforcing even if the /etc/selinux/config was altered.
 
+The version for CGroup in Mariner images can be enabled by using the `CGroup` key with value containing which version to use on boot. The value that can be chosen is either `version_one` or `version_two`. 
+The `version_two` value will set the cgroupv2 to be used in Mariner by setting the config value `systemd.unified_cgroup_hierarchy=1` in /boot/systemd.cfg. The value `version_one` or no value set will keep
+cgroupv1 (current default) to be enabled on boot. This KernelCommandLine option can be used to quickly generate Mariner images (including marketplace) with cgroupv2 enabled to fulfill customer needs.
```

Comment:
```suggestion
cgroupv1 (current default) to be enabled on boot.
```
nit - I think this sentence is not necessary to state since the purpose of `KernelCommandLine` is already stated earlier.